ebmbot/slack.py

```diff
@@ -12,7 +12,23 @@ def notify_slack(
     logger.info(
         "Sending message", channel=channel, message=message_text, thread_ts=thread_ts
     )
-    resp = slack_client.chat_postMessage(
-        channel=channel, thread_ts=thread_ts, **msg_kwargs
+    # In case of any unexpected transient exception posting to slack, retry up to 3
+    # times and then log the error, to avoid errors in scheduled jobs.
+    attempts = 0
+    while attempts < 3:
+        try:
+            resp = slack_client.chat_postMessage(
+                channel=channel, thread_ts=thread_ts, **msg_kwargs
+            )
+            return resp.data
+        except Exception as err:
+            attempts += 1
```

Comment:
Is is worth a small `sleep()` here? Might not be worth it, just wondered if you'd considered it.

Reply:
I did consider it; I guess for a maximum 3 sec delay it's probably worth adding